src/packagedcode/maven.py

```diff
@@ -973,12 +1001,7 @@ def parse(location=None, text=None, check_is_pom=True, extra_properties=None):
             # complex defeinition in Maven
             qualifiers['type'] = extension
 
-    # TODO: we join all data in a single text: this may not be right
-    declared_license = []
-    for lic in pom.licenses:
-        lt = (l for l in [lic['name'], lic['url'], lic['comments']] if l)
-        declared_license.extend(lt)
-    declared_license = '\n'.join(declared_license)
+    declared_license = get_maven_licenses(pom.licenses)
```

Comment:
What do you gain with this vs. using the plain and simple `declared_license = pom.licenses` ?

Reply:
Yep, I agree. We already have pom.licenses which is a list of ordered dict.